Kubernetes/flannel/start-kubeproxy.ps1

```diff
@@ -0,0 +1,22 @@
+Param(
+    [ValidateSet("l2bridge", "overlay",IgnoreCase = $true)] [parameter(Mandatory = $true)] $NetworkMode,    
+    [parameter(Mandatory = $false)] $LogDir = "C:\k",    
+    $NetworkName = "cbr0"
+)
+
+$networkName = $NetworkName.ToLower()
+ipmo c:\k\hns.psm1
+Get-HnsPolicyList | Remove-HnsPolicyList
+
+if ($NetworkMode -eq "l2bridge")
+{
+    c:\k\kube-proxy.exe --v=4 --proxy-mode=kernelspace --hostname-override=$(hostname) --kubeconfig=c:\k\config --network=$networkName --enable-dsr=false --log-dir=$LogDir --logtostderr=false
```

Comment:
isn't it ``--network-name``? kube-proxy doesn't recognize the ``--network`` argument, and you're also passing ``--network-name`` below for overlay.

Reply:
Thanks, I validated l2bridge now & fixed the typo.